stdlib/public/SDK/Foundation/Foundation.swift

```diff
@@ -567,11 +567,12 @@ extension Array : _ObjectiveCBridgeable {
 
 extension NSDictionary : ExpressibleByDictionaryLiteral {
   public required convenience init(
-    dictionaryLiteral elements: (NSCopying, AnyObject)...
+    dictionaryLiteral elements: (_Hashable, Any)...
```

Comment:
If we get generalized existentials at the same time, then we could just use `Hashable`.